Formula/docker-compose-completion.rb

```diff
@@ -1,8 +1,8 @@
 class DockerComposeCompletion < Formula
   desc "Completion script for docker-compose"
   homepage "https://docs.docker.com/compose/completion/"
-  url "https://github.com/docker/compose/archive/1.29.2.tar.gz"
-  sha256 "99a9b91d476062d280c889ae4e9993d7dd6a186327bafb2bb39521f9351b96eb"
+  url "https://github.com/docker/compose/archive/v2.0.1.tar.gz"
+  sha256 "5a1b1fdb9681c6f4b39fceab7d7dca12285e72cb55e3d35c31f4659cc939cd2b"
```

Comment:
assuming we should revert this?

Reply:
also, do we need to remove the livecheck block?

Reply:
Yes and yes

Reply:
> assuming we should revert this?

That is right, Thanks!!